.github/workflows/publish.yml

```diff
@@ -63,11 +68,41 @@ jobs:
       uses: gradle/actions/setup-gradle@e24011a3b5db78bd5ab798036042d9312002f252 # tag=v3
       with:
         arguments: ${{ matrix.test-type.arguments }}
+
+  # Build the docs-zip antora doc for the current branch, and upload generated docs-zip to workflow run.
+  # JDK21 is used because the antora plugin requires a JDK17 compatible version.
+  # Each deploy jobs can then download the docs-zip into ./docs/build/distributions/ in order to let it be included in published artifacts.
```

Comment:
```suggestion
  # Each deploy job can then download the docs-zip to ./docs/build/distributions/ in order to be included in published artifacts.
```

Reply:
fixed in https://github.com/reactor/reactor-core/pull/3806/commits/dfe2a219a32b03874e620f1d97f879acef81d969